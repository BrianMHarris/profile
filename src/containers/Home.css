
.hm-content {
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  flex-wrap: nowrap;
  width: 100%;
  height: 280px;
  padding: 25px 0 0 0;
  font-family: 'Poppins', serif;
  letter-spacing: 1px;
  color: var(--clr-txt-white);
  background-color: var(--clr-bg-header);
}

.hm-title-intro,
.hm-title-job {
  font-size: 1.1em;
}

.hm-title-name {
  font-size: 2.5em;
}

@media screen and (min-width: 361px) {
  .hm-title-intro,
  .hm-title-job {
    font-size: 1.25em;
  }

  .hm-title-name {
    font-size: 2.75em;
  }
}

@media screen and (min-width: 768px) {
  .hm-title-intro,
  .hm-title-job {
    font-size: 1.6em;
  }

  .hm-title-name {
    font-size: 3.25em;
  }
}
