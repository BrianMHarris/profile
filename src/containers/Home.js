import React, { Component } from 'react';
import "./Home.css"


class Home extends Component {
  render() {
    return (
      <div className="hm-content">
        <div id="home"></div>
        <div className="hm-title">
          <div className="hm-title-intro">
            Hi, my name is
          </div>
          <div className="hm-title-name">
            Brian Harris.
          </div>
          <div className="hm-title-job">
            I'm a Full Stack Web Developer
          </div>
        </div>
      </div>
    );
  }
}

export default Home;
