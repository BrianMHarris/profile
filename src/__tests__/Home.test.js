import React from 'react';
import ReactDOM from 'react-dom';
import Home from '../containers/Home';
import renderer from "react-test-renderer";

describe('Home', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Home />, div);
  });

  it("matches the snapshot", () => {
    var tree = renderer.create(
        <Home />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
