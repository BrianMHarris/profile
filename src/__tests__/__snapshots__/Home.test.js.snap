// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home matches the snapshot 1`] = `
<div
  className="hm-content"
>
  <div
    id="home"
  />
  <div
    className="hm-title"
  >
    <div
      className="hm-title-intro"
    >
      Hi, my name is
    </div>
    <div
      className="hm-title-name"
    >
      Brian Harris.
    </div>
    <div
      className="hm-title-job"
    >
      I'm a Full Stack Web Developer
    </div>
  </div>
</div>
`;
